.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,36 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
+// https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-existing-dockerfile
+{
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
```

Comment:
Minor: Comments in json are non-standard. I think they'll cause issues in any editor other than VSCode (the syntax highlighting on GitHub doesn't like them too). Can we reduce this to just the JSON, and add an adjacent README pointing at the docs if necessary?

Reply:
Sure, this was generated by VS Code, I'll scrub it.

Reply:
Done